### ISSUE TITLE
Release J-HMDB21 annotations

### ISSUE THREAD

**Issue by ZZZZZZZZJ** [wei-tim/YOWO#35]
Thank you for your sharing code! 

But I found there are no labels of J-HMDB 21 dataset in the link.

Could you release the J-HMDB21 labels like 'labels/video_name/img_idx.txt' (which is needed by the training code.)

So that we can run the training code of J-HMDB 21 directly!

**Comment by okankop** [comment]
You can refer to #25 for the jhmdb labels.

**Comment by ZZZZZZZZJ** [comment]
Thanks for the labels! I run the training procedure successfully.
But I have a new question that I **can not reproduce the J-HMDB results**.
After 25 epochs, I only get 65.07%, much lower than 74.4% shown in the paper.
```
AP: 98.33% (1)
AP: 98.74% (10)
AP: 97.78% (11)
AP: 39.40% (12)
AP: 40.06% (13)
AP: 94.73% (14)
AP: 71.47% (15)
AP: 30.06% (16)
AP: 31.94% (17)
AP: 14.99% (18)
AP: 45.34% (19)
AP: 22.74% (2)
AP: 71.97% (20)
AP: 64.42% (21)
AP: 91.01% (3)
AP: 76.60% (4)
AP: 94.07% (5)
AP: 57.49% (6)
AP: 56.25% (7)
AP: 69.93% (8)
AP: 99.21% (9)
mAP: 65.07%
```
And my jhmdb.cfg is 
```
[net]
batch = 10
clip_duration = 16
height = 224
width = 224
channels = 3
momentum = 0.9
decay = 0.0005

learning_rate = 0.0001
max_batches = 100000
steps = 10000,20000,30000,40000
scales = 0.5,0.5,0.5,0.5


[region]
anchors = 0.95878, 3.10197, 1.67204, 4.0040, 1.75482, 5.64937, 3.09299, 5.80857$
classes = 21
num = 5

object_scale = 5
noobject_scale = 1
class_scale = 1
coord_scale = 1
```
my run_jhmdb-21.sh is 
```
python train.py --dataset jhmdb-21 \
                --data_cfg cfg/jhmdb21.data \
                --cfg_file cfg/jhmdb21.cfg \
                --n_classes 21 \
                --backbone_3d resnext101 \
                --backbone_3d_weights weights/pretrained/resnext-101-kinetics-h$
                --freeze_backbone_3d \
                --backbone_2d darknet \
                --backbone_2d_weights weights/yolo.weights \
                --freeze_backbone_2d \
```
Does anything go wrong? Or if there are somethings unknown for me to influence the results?

**Comment by okankop** [comment]
Moat probably the model is overfitting. Do not wait till the end of 25 epoch to run validation. Best performance is usually achieved between 5-10 epochs.

**Comment by kinivi** [comment]
> Moat probably the model is overfitting. Do not wait till the end of 25 epoch to run validation. Best performance is usually achieved between 5-10 epochs.

Can I use negative examples for the train? For example some actions without labels (for custom dataset)?

**Comment by west-i** [comment]
@ZZZZZZZZJ 
Hi,
Could you share the J-HMDB21 labels to 3041622569@qq.com ?
Thanks!

**Comment by Holmes-GU** [comment]
@ZZZZZZZZJ Have you successfully reproduced the ucf24 results reported in the paper?

**Comment by Riiick2011** [comment]
> @ZZZZZZZZJ Have you successfully reproduced the ucf24 results reported in the paper?

Same question.
